PhpAmqpLib/Connection/AMQPConnectionConfig.php

```diff
@@ -324,6 +345,26 @@ public function setHeartbeat(int $heartbeat): void
         $this->heartbeat = $heartbeat;
     }
 
+    public function getHeartbeatTune(): int
+    {
+        return $this->heartbeatTune;
+    }
+
+    public function setHeartbeatTune(?int $heartbeatTune): void
+    {
+        if ($heartbeatTune === null) {
```

Comment:
why do we need nullable argument here?

Reply:
to avoid boilerplate referencing `AMQPConnectionConfig::HEARTBEAT_TUNE_DEFAULT` if no value is provided and you want to use the default. I'm also fine with forcing a value and removing the nullable, if you can confirm that's what you prefer let me know and I'll update the code

Reply:
I think null can be misleading here, as yet another possible option. Default value is set by default, so no need to set it.

Reply:
changed!